packages/mask/shared-ui/locales/en-US.json

```diff
@@ -382,10 +383,13 @@
     "plugin_trader_safety_alert": "Anyone can create and name any ERC20 token on Ethereum, including creating fake versions of existing tokens and tokens that claim to represent projects that do not have a token. Similar to Etherscan, this site automatically tracks analytics for all ERC20 tokens independent of token integrity. Please do your own research before interacting with any ERC20 token.",
     "plugin_trader_total_supply": "Total Supply",
     "plugin_trader_circulating_supply": "Circulating Supply",
-    "plugin_trader_volume_24": "Volume (24h)",
+    "plugin_trader_volume_24": "24 Hour Trading Vol",
```

Comment:
```suggestion
    "plugin_trader_volume_24": "24 Hours Trading Vol",
```

Reply:
`24 hour` is right, according to https://www.quora.com/Is-24-hours-right-or-24-hour

Reply:
IC